internal/core/core.go

```diff
@@ -323,7 +323,7 @@ func ComputeLineStartsSeq(text string) iter.Seq[TextPos] {
 		lineStart := 0
 		for pos < len(text) {
 			b := text[pos]
-			if b < 0x7F {
+			if b < utf8.RuneSelf {
```

Comment:
This is a behavior change, but a correct one if you read the original code (or think about the logic).